core/src/main/java/org/apache/iceberg/BaseMetastoreCatalog.java

```diff
@@ -278,84 +268,6 @@ private Transaction newReplaceTableTransaction(boolean orCreate) {
     }
   }
 
-  /**
-   * Drops all data and metadata files referenced by TableMetadata.
-   * <p>
-   * This should be called by dropTable implementations to clean up table files once the table has been dropped in the
-   * metastore.
-   *
-   * @param io a FileIO to use for deletes
-   * @param metadata the last valid TableMetadata instance for a dropped table.
-   */
-  protected static void dropTableData(FileIO io, TableMetadata metadata) {
```

Comment:
Because this was `protected`, I think we should keep it and just delegate to where the method moved. We can also deprecated it and set a version when it will be removed:

```
  /**
   * ...
   * @deprecated will be removed in 0.11.0; use CatalogUtil.dropTableData instead.
   */
```

Reply:
Good point!
Done